plugins/dev-tools/src/test_helpers.mocha.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as commonHelpers from './common_test_helpers.mocha';
+import * as fieldHelpers from './field_test_helpers.mocha';
+
+const {Run} = commonHelpers;
```

Comment:
You're only using one thing from `commonHelpers`, so you can just do

`import {Run} from './common_test_helpers.mocha';`

Reply:
I expect for this deconstruction to become larger as more common helpers are added.
I'm already planning on also adding to the list runTestCases in/for another PR I'm working on.
So, I think it makes sense to leave it like this for this reason.